src/schemas/f142/f142_rw.cxx

```diff
@@ -596,8 +592,8 @@ int32_t HDFWriterModule::flush() { return 0; }
 
 int32_t HDFWriterModule::close() { return 0; }
 
-HDFWriterModuleRegistry::Registrar
-    g_registrar_HDFWriterModule("f142", HDFWriterModule::create);
+HDFWriterModuleRegistry::Registrar<HDFWriterModule>
+    RegisterEPICSDataWriterModule("f142");
```

Comment:
Should be `static`.

Naming, like in the above comment?

Reply:
See previous comment.